code/_onclick/hud/alert.dm

```diff
@@ -666,8 +666,7 @@ so as to remain in compliance with the most up-to-date laws."
 		return usr.client.Click(master, location, control, params)
 
 /atom/movable/screen/alert/Destroy()
-	. = ..()
 	severity = 0
-	master = null
```

Comment:
Why are you dropping this?

Reply:
There's `master = null` at `/atom/movable/screen/Destroy()`